### PR TITLE
XWIKI-10850: The "Create new group" field is not consistent with the …

### PR DESCRIPTION
**Pull request by gabrielasmeria**
…"Create group" and "Cancel" buttons

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/lightbox/lightbox.css [hunk 1]
```diff
@@ -67,7 +67,7 @@
 #lb {
   left: 0;
   margin-left: 0;
-  width: auto;
+  width: 100%;
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/lightbox/lightbox.css]: Why is this needed? The value was reset in order to fix XWIKI-10734. 
Would be nice to see how it affects the responsiveness of Modals.

Also I think width:auto rule should have been removed, since that's the default.

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/addnewgroup.vm [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+<form method="post" onsubmit="return false;" class="xform">
+<dl>
+<dt>
+<label for="newgroupi">$services.localization.render('rightsmanager.createnewgroup')</label>
+</dt>
+<dd>
+<input type="text" name="newgroup" value="" id="newgroupi">
+</dd>
+</dl>
+<p>
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/addnewgroup.vm]: "div.buttons" could have been used. 

To be honest, I'm not sure if we should use the: 
- vertical form standard;
- bootstrap standard; 
- or both (.button + .btn classes),
since we still haven't agreed on the standard. 

From a consistency point of view and until we reach a conclusion I prefer the approach you took (the vertical form standard), especially since we extend on the old classes.

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/lightbox/lightbox.css (hunk 1 above)
Comment (gabrielasmeria) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/lightbox/lightbox.css]: I had to change it to 100%, because if I would've left it auto, the modal was occupying just 50-60% of the screen. I tested the modals from the screenshots from XWIKI-10734 and they're all working.

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/addnewgroup.vm (hunk 2 above)
Comment (gabrielasmeria) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/addnewgroup.vm]: I've added div.buttons. Thanks!
